src/debug.ts

```diff
@@ -119,15 +119,12 @@ export class RubyConfigurationProvider extends ConfigurationProvider
     }
 }
 
-export class OrogenConfigurationProvider extends ConfigurationProvider
+export class OrogenConfigurationProvider extends CXXConfigurationProvider
```

Comment:
How do I "auto" start and configure the task?

Reply:
Not sure what you are asking ...

Reply:
It would be nice if I didn't have to manually start and configure the task while debugging. The previous implementation (and also the one that used `rock-run`) provided a "start" property that would automatically start the task once it was reachable and "confDir" would set  `Orocos.conf.load_dir` to load configuration files

Reply:
I meant to keep them, will fix package.json